guides/doc-Upgrading_and_Updating/topics/upgrading_a_disconnected_satellite.adoc

```diff
@@ -265,7 +265,7 @@ In such a case, change directory, for example to the *_root_* user's home direct
 ====
 +
 If the script fails due to missing or outdated packages, you must download and install these separately.
-For more information, see the https://access.redhat.com/documentation/en-us/red_hat_satellite/{AccessRedHatComVersion}/html-single/installing_satellite_server_from_a_disconnected_network/installing-satellite-server-disconnected#resolving-package-dependency-errors_satellite[Resolving Package Dependency Errors] section in the _Installing {ProjectServer} from a Disconnected Network_ guide.
+For more information, see the {BaseURL}/installing_satellite_server_from_a_disconnected_network/installing-satellite-server-disconnected#resolving-package-dependency-errors_satellite[Resolving Package Dependency Errors] section in the _Installing {ProjectServer} from a Disconnected Network_ guide.
```

Comment:
Does this for all all projects? IMHO this would need a "InstallingServerDisconnectedDocURL" attribute.

Reply:
I'm not too familiar with the conventions. Is that something we could agree on and write down somewhere? That would also be a good thing for people new to this repository.

Reply:
No convention, but there's already a small list; basically each guide has its own `DocURL` attribute: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/attributes-base.adoc#L1

IMHO: one directory in `guides/` == one attribute.

Reply:
I think that makes sense. Especially since in upstream we need to take the build into account (foreman-el, foreman-deb, katello) which is reflected in the index name.